src/pwncore/routes/team.py

```diff
@@ -20,7 +25,12 @@ async def team_login():
     return {"status": "logged in!"}
 
 
+# Unable to test as adding users returns an error
```

Comment:
The model `User.save` method has a bug: It is not checking whether `members` fields exists on the Team before trying to filter for it.

In `src/pwncore/models/user.py`, change:
```py
- if self.team is not None:
+ if self.team is not None and hasattr(self.team, "members"):
```